neclib/coordinates/observations/optical_pointing.py

```diff
@@ -21,16 +23,62 @@ def readlines_file(self, filename: str) -> List[str]:
         return contents
 
     def _catalog_to_pandas(self, catalog_raw: List[str]):
-        # TODO: Implement.
-        ...
+        name_data = []
+        ra_data = []
+        dec_data = []
+        multiple_data = []
+        vmag_data = []
+        pmra_data = []
+        pmdec_data = []
+        az_data = []
+        el_data = []
+        for line in catalog_raw:
+            try:
+                name = line[7:14]
+                ra = (
+                    float(line[75:77]) * u.hourangle
+                    + float(line[77:79]) * u.arcmin
+                    + float(line[79:83]) * u.arcsec
+                ).to(u.deg)
```

Comment:
- この式のままだとminとsecは15のFactor分だけ小さい値を返す結果を得ます。

例：
```python
>>> (11*u.hourangle+31*u.arcmin+24.2*u.arcsec).to(u.deg)
<Quantity 165.52338889 deg>

>>> 11*15+31*15/60+24.2*15/3600
172.85083333333333 # 本来の値

>>> from astropy.coordinates import Angle
>>> Angle("11h31m24.2s").to(u.deg)
<Angle 172.85083333 deg>
```

- 例えば以下のように`astropy.coodinates.Angle`を利用して修正してください。


```suggestion
from astropy.coordinates import Angle
...
                ra_raw = line[75:77]+"h"+line[77:79]+"m"+line[79:83]+"s"
                ra = Angle(ra_raw).to(u.deg)
```

Reply:
修正しました